package.json

```diff
@@ -40,7 +40,8 @@
     },
     "modulePathIgnorePatterns": [
       "<rootDir>/templates/"
-    ]
+    ],
+    "globalSetup": "<rootDir>/globalSetup.js"
```

Comment:
I don’t know what the difference is 🤔  but have you tried to setup it in `jest.setup.js`?

Reply:
From [here](https://jestjs.io/docs/en/configuration), I see the difference is that `globalSetup` is run once at the beginning while `setupFiles` is run before each test. But setting `process.env.TZ` in `jest.setup.js` isn't working. I am not sure why :thinking:.